README.rst

```diff
@@ -77,30 +77,7 @@ If you have questions or have trouble using the app please file a bug report at:
 https://github.com/jazzband/django-axes/issues
 
 
-Contributions
+Contributing
 -------------
```

Comment:
The divider line is too short and rST bakery might fail, could you adjust it as well?

Reply:
Fixed)